History/js/new.js

```diff
@@ -149,85 +149,14 @@ var callbacks = {
     
     var tableBody = document.getElementById("table_body");
     for(var i = 0; i < response.json.length; i++) {
-
-      var href = response.json[i].privly_URL;
-      var app = response.json[i].privly_application;
-
-      // Rename deperecated apps
-      if ( app === "ZeroBin" ) {
-        app = "Message";
-      }
```

Comment:
We can remove this in a few weeks, but we need it at the moment -- I just need to find time to write and test a migration server side.